ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/QueryDescription.java

```diff
@@ -68,6 +74,11 @@ public QueryDescription(
     this.overriddenProperties = ImmutableMap.copyOf(Objects
         .requireNonNull(overriddenProperties, "overriddenProperties"));
     this.state = Objects.requireNonNull(state, "state");
+    
+    this.ksqlHostQueryState =
+        new TreeMap<>(Comparator.comparing(KsqlHostInfoEntity::toString));
+    this.ksqlHostQueryState
+        .putAll(Objects.requireNonNull(ksqlHostQueryState, "ksqlHostQueryState"));
```

Comment:
Would be good to add a unit test to _test_ this is returning an ordered collection. (Or just JSON with ordered fields).

Reply:
I actually only wanted an ordered set because some of the new tests I added weren't working because the `toString()` of this map would return different results, but those tests don't rely on the string representation anymore since the `state` field was removed from the constructor so order doesn't matter for now.